packages/vuetify/src/components/VBtnGroup/VBtnGroup.tsx

```diff
@@ -16,8 +16,15 @@ import { makeVariantProps } from '@/composables/variant'
 import { toRef } from 'vue'
 import { genericComponent, propsFactory, useRender } from '@/util'
 
+// Type
+import type { PropType } from 'vue'
+
 export const makeVBtnGroupProps = propsFactory({
   divided: Boolean,
+  direction: {
+    type: String as PropType<'horizontal' | 'vertical'>,
```

Comment:
This might be have a better dev experience if vertical was a boolean. Spelling out props and attributes ripe for spelling bugs; thoughts?

Reply:
I see your point. But for the sake of uniformity,  have implemented the props as utilized in VTimeline, VWindow, VSlideGroup and other Vuetify components.

Reply:
> This might be have a better dev experience if vertical was a boolean. Spelling out props and attributes ripe for spelling bugs; thoughts?

I'd normally agree with you but it's too far out for that type of change. With that being said, I'm not sure why we didn't make it boolean.